setup.py

```diff
@@ -104,7 +104,8 @@ def _collect_requirements(requires):
         "sphinxcontrib-mermaid~=0.7.1",
         "myst-parser~=1.0.0",
         "Jinja2<3.1.0",
-        "kedro-datasets[all]~=1.4.0",
+        # "kedro-datasets[all]~=1.4.0",
```

Comment:
Is there a reason you've retained the old line (leaving it commented) when replacing it?